Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Inferno",
+    platforms: [.iOS(.v15), .macOS(.v12), .macCatalyst(.v15), .tvOS(.v15), .visionOS(.v1)],
```

Comment:
Using the APIs requires iOS 17 (etc), but these are the minimum version that can compile the included shaders because they include Metal v2.4 https://developer.apple.com/documentation/metal/mtllanguageversion/version2_4